git-merge-conflict.md

```diff
@@ -0,0 +1,52 @@
+
+## Instructions
+### Baseline Setup
+Before we can start putting the recipe together we need to make sure that everyone in your group is working from the same starting point and can effectively contribute to the recipe building effort.
+
+1. **Split into groups**: Each table is marked with a particular group number and everyone at that table will be part of the group. There should be six people in each group.
+1. **Fork the activity repo**: One person in your group should fork the [carnival-recipes](https://github.com/Ada-C7/carnival-recipes) repo and then add everyone else as a collaborator.
+1. **Clone the forked repo**: Everyone else in your group should clone the forked repository.
+1. **Create the recipe file**: To keep things better organized, one person in your group should take the [title scrap](https://github.com/Ada-C7/carnival-recipes/#recipe-scraps-for-each-group) and copy its contents into another file in the `recipes/` folder. This new file should be named appropriately based on the recipe title. **You should also delete the title scrap once its contents have been moved into the recipe file.**
+1. **Pull the recipe file**: Once the recipe file has been created and committed, everyone else in your group should `git pull` that commit to their local copy of the repository.
+1. **Find your recipe scrap**: Each group is assigned six recipe scraps that should all be part of the same recipe. You can split these scraps up amongst your group however you wish. The scraps assigned to each group are [listed in the carnival-recipes repo](https://github.com/Ada-C7/carnival-recipes/#recipe-scraps-for-each-group).
+1. **Add your recipe scrap**: Copy the contents of your recipe scrap into the recipe file and delete the recipe scrap then commit both of these changes.
+
+**Do not** push your changes to GitHub yet! If you're finished early with this step help your groupmates.
+
+### Building the recipe
+Once everyone in your group has completed the baseline setup instructions above you're ready to start building the recipe together. There are a number of ways that your group can accomplish that, and you should discuss them to figure out which strategy to employ:
+
+- **Round robin**: This strategy involves direct coordination between everyone in the group, resulting in a strictly ordered merge sequence.
+  - One at a time each person pushes their changes and resolves any merge conflicts.
+  - The next person in turn pulls those changes and pushes their own until everyone's changes have been pushed.
+- **Ad-hoc**: This approach relies the most upon Git's merge coordination capabilities to ensure that everyone's changes get merged in properly.
```

Comment:
@kari I'm finding it difficult to work out how to suggest that a single team use more than one strategy. Did you mean have two teams do the branches and everyone else does ad-hoc merging?

Having a single team employ multiple strategies could work fine if we have six groups of eight people, though.